examples/python/awq-quantized-model.md

```diff
@@ -1,10 +1,16 @@
 # Create AWQ-quantized and optimized ONNX models from PyTorch models with AutoAWQ + ONNX Runtime generate() API
 
 ## Steps
-1. [Download your PyTorch model](#1-download-your-pytorch-model)
-2. [Install AutoAWQ](#2-install-autoawq)
-3. [Install the generate() API](#3-install-the-generate-api)
-4. [Run example script](#4-run-example-script)
+- [Create AWQ-quantized and optimized ONNX models from PyTorch models with AutoAWQ + ONNX Runtime generate() API](#create-awq-quantized-and-optimized-onnx-models-from-pytorch-models-with-autoawq--onnx-runtime-generate-api)
```

Comment:
Can we revert this and just add the sub-sections for item 3 (CPU, CUDA, and DML)?

```md
1. [Download your PyTorch model](#1-download-your-pytorch-model)
2. [Install AutoAWQ](#2-install-autoawq)
3. [Install the generate() API](#3-install-the-generate-api)
   - [CPU](#cpu)
   - [CUDA](#cuda)
   - [DirectML)(#directml)
5. [Run example script](#4-run-example-script)
```